cdn/cdnrefresh.php

```diff
@@ -36,7 +36,7 @@ function generate_hashed_version_file($source, $deploy, $source_file) {
       $dest_file = $source_file_short;
     }
     if(!file_exists($deploy . $dest_file)) {
-      echo $source_file . " => " . $deploy . $dest_file . "\n";
+      echo $source_file . " => " . $deploy . $dest_file . "<br/>";
```

Comment:
This is a CLI process, why emit html?

Reply:
I used https://github.com/keymanapp/keymanweb.com/blob/0dba6472d3300ef15a4820ee53953d0051333aa0/cdn/cdnrefresh.php#L39 as reference

Reply:
Reverted to "\n"

Reply:
gotcha